common.go

```diff
@@ -89,17 +90,30 @@ func (term *dbTerm) addTags(tags ...string) {
 	term.Tags = appendStringUnique(term.Tags, tags...)
 }
 
+func (term *dbTerm) addTermTags(termTags ...string) {
+	term.TermTags = appendStringUnique(term.TermTags, termTags...)
+}
+
 func (term *dbTerm) addRules(rules ...string) {
 	term.Rules = appendStringUnique(term.Rules, rules...)
 }
 
+func (term *dbTerm) crushTags() string {
+	tags := strings.Join(term.Tags, " ")
+	if len(term.TermTags) == 0 {
+		return tags
+	} else {
+		return tags + "\t" + strings.Join(term.TermTags, " ")
```

Comment:
Is the idea that `tags` and `termTags` are separated by a tab and then the extension decides what is which is which at runtime? I think it would be cleaner to add an actual field to the export (similar to Sequence). The output of `yomichan-import` should be ready for consumption by the extension and there should not be any additional parsing taking place after that.

Reply:
Yes, I decided for this to enable easier backwards compatibility with dictionaries exported with older version of yomichan-import. This is what's done in Yomichan: https://github.com/siikamiika/yomichan/commit/4fb983a70cd02c0c0126a26e70d40261d5b2a51d. If a new field for termTags is added, should it come after tags or at the end like sequence?

Reply:
Yeah, adding a field at the end will allow us to keep compatibility since older versions of Yomichan will just ignore it when destructuring the array.